services/fxaccounts/FxAccountsKeys.jsm

```diff
@@ -71,7 +79,31 @@ class FxAccountsKeys {
    *        or null if no user is signed in
    */
   async getKeys() {
-    return this._fxia.withCurrentAccountState(async currentState => {
+    if (this._fxia.rustFxa) {
+      let {
+        key: { kid, k: kSync },
+      } = await this._fxia.rustFxa.getAccessToken(SCOPE_OLD_SYNC);
+      let kXCS = kid.split("-")[1];
```

Comment:
Yikes! I understand why we're doing this, but I wonder if we could do the inverse refactoring here instead - rather than carving up the scoped-keys data held by the rust component so we can return it in the older format expected by desktop, could we refactor the desktop code that is calling this so that it instead works with scoped keys? I think @vladikoff's work to access sync via OAuth headed a bit in this direction already.